filament/backend/include/backend/TargetBufferInfo.h

```diff
@@ -53,7 +53,11 @@ class TargetBufferInfo {
 };
 
 class MRT {
-    TargetBufferInfo mInfos[4];
+public:
+    static constexpr int NUM_TARGETS = 4;
```

Comment:
usually we use the `FOO_COUNT` convention instead of `NUM_FOO`.

Reply:
Fixed.